### PR TITLE
Feature/#75

### PR DESCRIPTION
**Pull request by ymarkoff**
#44

### CODE REVIEW

**Review by winniepukki — CHANGES_REQUESTED** [review]
@ymarkoff fix branch conflicts

**Comment by winniepukki** [comment]
@ymarkoff why are the branching and commit strategies ignored, again???

**Comment by winniepukki** [comment]
@ymarkoff why was the manual hook implementation ignored? what is the need for installing the package??

**Comment by ymarkoff** [comment]
> @ymarkoff why are the branching and commit strategies ignored, again???

I'll make a new branch, I thought that I based this one on Staging, apologies.

**Comment by winniepukki** [comment]
@ymarkoff please use: `feature/#<issue-nr>` for calling branches, and `feature/#<issue-nr> – <comment>` for commit messages. It is complicated to track changes and reference issues

**Comment by winniepukki** [comment]
> I'll make a new branch, I thought that I based this one on Staging, apologies.

There is no need for a new branch, just change the target PR branch

**Comment by ymarkoff** [comment]
> @ymarkoff please use: `feature/#<issue-nr>` for calling branches, and `feature/#<issue-nr> – <comment>` for commit messages. It is complicated to track changes and reference issues

Noted down.

File: hooks/useMobileDetector/useMobileDetector.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
...
+    }
+
+    return isMobile;
```
Comment (winniepukki) [inline hooks/useMobileDetector/useMobileDetector.js:8]: Please use the imperative programming:
```js
import { useState, useEffect } from "react";

export const useDeviceDetect = () => {
  const [isMobile, setMobile] = useState(false);

  useEffect(() => {
    const userAgent = typeof window.navigator === "undefined" ? "" : navigator.userAgent;
    const mobile = !!userAgent.match(
      /Android|BlackBerry|iPhone|iPad|iPod|Opera Mini|IEMobile|WPDesktop/i
    );
    setMobile(mobile);
  }, []);

  return { isMobile };
};

export default useDeviceDetect;
```

**Review by winniepukki — CHANGES_REQUESTED** [review]
@ymarkoff please explain your choice for using such a variety of operating systems

**Review by winniepukki — APPROVED** [review]
@ymarkoff good. not merging, for now, considering Cypress is not good for testing the mobile environment. Leave it to me

**Pull request closed without merging**